tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "paths": {
       "puppeteer": ["node_modules/@types/puppeteer"]
     },
+    "skipLibCheck": true,
```

Comment:
This avoids that `npm run build break`, but isn't great. Sadly, we can't dependably use the prior puppeteer version types in our package.json files as they tend to change version-to-version and don't work properly.